source/_integrations/recorder.markdown

```diff
@@ -306,6 +306,21 @@ If you want to use Unix Sockets for PostgreSQL you need to modify the `pg_hba.co
 
 </div>
 
+<div class='note warning'>
+
+If your database name, user or password contains special characters like brackets, you first need to URL encode them to avoid potential issues.
+You can for example open a Python3 console and type:
+
+```python
+import urllib.parse
+urllib.parse.quote("the[password")
+```
```

Comment:
Uh? Let's not put code here. Ever.

Additionally, this is an URL so, uh yeah 🤷